wgengine/filter/filter.go

```diff
@@ -107,35 +107,28 @@ func New(matches Matches, shareStateWith *Filter) *Filter {
 func maybeHexdump(flag RunFlags, b []byte) string {
 	if flag != 0 {
 		return packet.Hexdump(b) + "\n"
-	} else {
-		return ""
 	}
+	return ""
```

Comment:
Style nitpick: I would invert these two conditions. The straight line down the function should be the "interesting" codepath, with "uninteresting exits" (e.g. errors, "skip this because it's turned off") in branches. In this case the interesting outcome is a hexdump, so it should be the straight line path through the function.

Doesn't matter much for this tiny and obvious helper, but it's a life saver in larger funcs.

Reply:
Yep it does pass! And I made the change for this one.